packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -437,6 +482,9 @@ describe('HttpHandler', () => {
       .to(createUnexpectedHttpErrorLogger());
     rootContext.bind(SequenceActions.SEND).to(writeResultToResponse);
     rootContext.bind(SequenceActions.REJECT).toProvider(RejectProvider);
+    rootContext
+      .bind(RestBindings.ERROR_HANDLER_OPTIONS)
+      .to(defaultErrorHandlerOptions);
```

Comment:
I think it would be nicer to have `RestBindings.ERROR_HANDLER_OPTIONS` as an optional binding, so that the tests don't have to provide it. Not a big deal though.